src/cmds/qsub.c

```diff
@@ -162,207 +162,179 @@
 
 #define MAX_QSUB_PREFIX_LEN 32
 #define QSUB_DMN_TIMEOUT_LONG 60  /* timeout for qsub background process */
-#define QSUB_DMN_TIMEOUT_SHORT 5  
+#define QSUB_DMN_TIMEOUT_SHORT 5
 
-static char PBS_DPREFIX_DEFAULT[] = "#PBS";
 
-/* globals */
+/* extern global variables */
 #ifdef NAS /* localmod 005 */
 extern void set_attr_resc(struct attrl **attrib, char *attrib_name, char *attrib_resc, char *attrib_value);
 #endif /* localmod 005 */
 extern char *msg_force_qsub_update;
-int comm_sock;
 
-/* Socket for x11 communication */
-int X11_comm_sock;
+/*
+ * The following variables are file static (ie. global in this file).
+ */
+static char PBS_DPREFIX_DEFAULT[] = "#PBS";
+static char	const pbs_o_env[] = "PBS_O_"; /* prefix for environment variables created by qsub */
 
-/* Max size of buffer to store Xauth cookie length */
-#define XAUTH_LEN   512
+/* Warning/Error messages */
+#ifdef WIN32
+static char const intergui_warn[] = "qsub: only interactive jobs can have GUI\n";
+#endif
+static char const interblock_warn[] = "qsub (Warning) : setting \"block\" attribute as \"true\""
+	" for an interactive job will not return job's exit status\n";
+static char const interarray[] = "qsub: interactive and array job submission cannot be used together\n";
+static char const norerunarray[] = "qsub:  cannot submit non-rerunable Array Job\n";
+static char const reruninteract[] = "qsub (Warning): Interactive jobs will be treated as not rerunnable\n";
+static char const badw[] = "qsub: illegal -W value\n";
 
-/* Max size of buffer to store port information as string */
-#define X11_PORT_LEN 8
+/* Security library variables */
+static int	cs_init = 0;	/*1==security library initialized, 0==not initialized*/
+static int 	cred_type = -1;
```

Comment:
There is a [space][tab] combination on this line. Please see other comments related to aligning variable declarations. There are several instances where this occurs. Might as well clean them up since you are refactoring.

I also found at least one instance of [tab][space] that wasn't associated with a comment.